quickstarts/aws/amazon-codebuild/config.yml

```diff
@@ -0,0 +1,31 @@
+slug: amazon-codebuild
+description: |-
+  ## What is Amazon CodeBuild?
+  AWS CodeBuild is a fully managed build service that compiles your source code, runs unit tests, and produces artifacts that are ready to deploy. CodeBuild eliminates the need to provision, manage, and scale your own build servers.
+  ### Get started!
+  Start monitoring Amazon CodeBuild by connecting Amazon Web Services (AWS) to New Relic! Take a look at our documentation for information on how to set up this integration.
+  ### More info
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-codebuild-monitoring-integration/) to learn more about New Relic monitoring for Amazon Codebuild.
+summary: |-
+  Monitor Amazon CodeBuild by connecting AWS to New Relic
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title: Amazon CodeBuild
+documentation:
+  - name: Amazon CodeBuild installation docs
+    description: |
+      Monitor Amazon CodeBuild by connecting AWS to New Relic.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-codebuild-monitoring-integration/
```

Comment:
This link is broken and doesn't lead anywhere

Reply:
@brammerl , this PR has dependency on https://github.com/newrelic/docs-website/pull/14152/files. Once this is merged link will start working.